src/components/Questionaire/Header.tsx

```diff
@@ -25,45 +25,48 @@ const buttonStyle = {
   IconStyle: {
     pr: 2,
   },
+
 };
 
-const Header: React.FC<Props> = ({}) => {
+const Header: React.FC<Props> = ({ }) => {
```

Comment:
If we are not passing anything yet from the Props kindly remove the Props and just make it from `= ({}) =>` to `= () =>`